tutorials/event_sampler_tutorial.ipynb

```diff
@@ -0,0 +1,582 @@
+    "\n",
+    "First of all, let's define a [`Sky model`](https://docs.gammapy.org/dev/notebooks/models.html) for a point-like source centered 0.5 deg far from the Galactic Center and with a power-law spectral shape. We then save the model into a yaml file."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "spectral_model_pwl = PowerLawSpectralModel(index=2,amplitude=\"1e-12 TeV-1 cm-2 s-1\", reference='1 TeV')\n",
+    "spatial_model_point = PointSpatialModel(lon_0= \"0 deg\", lat_0=\"0.5 deg\", frame='galactic')\n",
+    "sky_model_pntpwl = SkyModel(spectral_model=spectral_model_pwl, spatial_model=spatial_model_point)\n",
+    "\n",
+    "models_pntpwl = Models([sky_model_pntpwl])\n",
+    "\n",
+    "file_model = \"point-pwl.yaml\"\n",
+    "models_pntpwl.write(file_model, overwrite=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Create the dataset\n",
+    "In this section we show how to create the `Dataset` object needed for the event sampling (for more info about to create `Dataset` objects, please visit this [link](https://docs.gammapy.org/dev/notebooks/analysis_2.html#Preparing-reduced-datasets-geometry)). \n",
+    "\n",
+    "Hereafter, we select the IRF from the South configuration of the CTA DC1 to simulate one observation. We set the pointing position of the simulated field at the Galactic Center and we fix the exposure time to 8 hr. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "IRF_FILE = \"$GAMMAPY_DATA/cta-1dc/caldb/data/cta/1dc/bcf/South_z20_50h/irf_file.fits\"\n",
+    "\n",
+    "POINTING = SkyCoord(0.0, 0.0, frame=\"galactic\", unit=\"deg\")\n",
+    "LIVETIME = 8 * u.hr"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We then define the energy axes (true and reconstruncted), the migration axis and the geometry of the observation. \n",
+    "\n",
+    "*This is a crucial point for the correct configuration of the event sampler. Indeed the spatial and energetic binning should be treaten carefully and the finer the better. For this reason, we suggest to define the energy axes by setting a minimum binning of least 10-20 bins per decade for all the source of interest. The spatial binning may instead be different from source to source and, at first order, it should be adopted a binning significantly smaller than expected source size.*\n",
+    "\n",
+    "For the examples that will be shown hereafter, we set the geometry of the dataset to a field of view of 4degx4deg and we  bin the spatial map with pixels of 0.02 deg."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# dataset config\n",
+    "ENERGY_AXIS = MapAxis.from_energy_bounds(\"0.1 TeV\", \"100 TeV\", nbin=10, per_decade=True)\n",
```

Comment:
Is the `e_reco` binning critical for the event sampling?

Reply:
Yes, it should be very well sampled.